### PR TITLE
Get member subnet id method mod

### PR DESCRIPTION
**Pull request by KingDaemonX**
mod : removed the unused service config argument and modified the test to fit new function modification

### CODE REVIEW

File: pkg/openstack/loadbalancer_test.go
Comment (pierreprinetti) [inline pkg/openstack/loadbalancer_test.go:1301]: Take a step back and look at what happens here. Your assertions are inside an `if` block. The `if` block determines which assertions will be run based on the content of **`err`**. Sometimes you need to have different assertions based on the expected values... but based on the _actual_ values? That looks suspicious.

Now look closer at what happens line by line, while you keep the variable values in your head. Imagine you expect `(tt.want == "", tt.wantErr == io.EOF)`, and your function returns `(got == "", err == nil)`. What would happen in this case?

File: pkg/openstack/loadbalancer.go
Comment (pierreprinetti) [inline pkg/openstack/loadbalancer.go:1436]: I'll defer to the maintainers to chime in on this change; `svcConf` is indeed unused, but I don't know why it was there in the first place.

CC @jichenjc @kayrus

**Review by pierreprinetti — CHANGES_REQUESTED** [review]

File: pkg/openstack/loadbalancer_test.go
Comment (KingDaemonX) [reply pkg/openstack/loadbalancer_test.go:1301]: Don’t mind that part 
I will change it, I don’t know if to tag the PR a draft or suggestion to modify the function 
I know it can’t be merged yet because i will need to rebase and restructure it if the modification

Comment (KingDaemonX) [reply pkg/openstack/loadbalancer_test.go:1301]: I made the push before your correction in that other PR and I was waiting for a review to know if the function modification looks good 
I’d have to close the PR if the removal of the `serviceConfig` isn’t authorised since there is an existing test already for the function and complete the entire modification when it is authorised 

so I apologize for the repetition, it wasn’t intentional as I have learnt the best usecase already from you and will not intentionally repeat same mistake twice

Comment (pierreprinetti) [reply pkg/openstack/loadbalancer_test.go:1301]: A good workflow may be:
1. Submit a PR
2. Make the required edits until you get a LGTM
3. Rebase
4. you'll get a new LGTM
5. wait for approval
6. merge

Conflicts are extraordinarily frequent these days, because several people are submitting PRs on the same test files. That's not a normal situation, and we have to bear with that for now.

Comment (pierreprinetti) [reply pkg/openstack/loadbalancer_test.go:1301]: In general, you can defer rebasing until the very last moment.

Comment (pierreprinetti) [reply pkg/openstack/loadbalancer_test.go:1301]: You don't have to turn your PR into WIP, unless you realise that it is NOT ready for review and you want to avoid wasting the reviewer's time. This is NOT the case here: your PR is tidy and definitely ready for reviews.

I have asked you to turn another PR of yours to WIP, just because you mentioned that it is not ready for review.

File: pkg/openstack/loadbalancer.go
Comment (dulek) [reply pkg/openstack/loadbalancer.go:1436]: I've took a quick look at the `git blame`, looks like it was there by mistake:

**Pull request closed without merging**
